build_local_windows.ps1

```diff
@@ -118,6 +118,9 @@ Set-AWSCredential -ProfileName aws_s3_windows;
 Set-DefaultAWSRegion -Region us-west-2;
 
 # Upload Tar Ball to dcos.download.io
+
 Write-S3Object -BucketName "downloads.dcos.io" -Key "dcos\testing\$env:TEAMCITY_BRANCH\windows\dcos_generate_config_win.sh" -File ".\dcos_generate_config_win.sh" -CannedACLName public-read -Metadata @{"Cache-Control" = "no-cache"};
+Write-S3Object -BucketName "downloads.dcos.io" -Key "dcos\testing\$env:TEAMCITY_BRANCH\commit\$env:BUILD_VCS_NUMBER\windows\dcos_generate_config_win.sh" -File ".\dcos_generate_config_win.sh" -CannedACLName public-read -Metadata @{"Cache-Control" = "no-cache"};
```

Comment:
Just for consistency, this is the no-cache header value used in Enterprise:
https://github.com/mesosphere/dcos-enterprise/blob/master/build_local_windows.ps1#L136

We could use the same here, and this PR can be merged.